bundles/org.openhab.binding.unifi/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -8,7 +8,7 @@
 
 		<label>UniFi Controller</label>
 		<description>A UniFi controller</description>
-
+		<semantic-equipment-tag>WirelessAccessPoint</semantic-equipment-tag>
```

Comment:
This is the controller which is the API endpoint used as bridge for all child Things.

Reply:
Agreed. This should be a `NetworkAppliance`

Reply:
Really? It's a piece of software. I have it running in a Docker instance on my NAS. Some have it running as a cloud service.

Reply:
> It's a piece of software.

I have it running on a Unifi Cloud Key (appliance)

> I have it running in a Docker instance on my NAS

Which is an appliance.

> Some have it running as a cloud service.

Which isn't.

=> So what is the answer?

Reply:
Removed.